pytorch/inference/docker/1.10/py3/Dockerfile.e3.graviton

```diff
@@ -130,8 +130,8 @@ RUN useradd -m model-server \
  && mkdir -p /home/model-server/tmp /opt/ml/model \
  && chown -R model-server /home/model-server /opt/ml/model
 
-COPY torchserve-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
-COPY config.properties /home/model-server
+COPY torchserve-e3-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
+COPY config.properties /home/model-server/config.properties
```

Comment:
do we need additional `/config.properties`?

Reply:
I saw some other places wrote it explicitly. in case the original file name change, we can keep docker file name consistant.